### ISSUE TITLE
Poission L-shape example, how to import training data

### ISSUE THREAD

**Issue by cauez** [lululxvi/deepxde#32]
Hi! 
I was analyzing the Poission example for L-shape, but could not figure out how the training data is imported to the model.

Can you please explain how the import of training data works in deepxde? And in which code lines does it happen?

**Comment by lululxvi** [comment]
In the line
```
data = dde.data.PDE(geom, pde, bc, num_domain=1200, num_boundary=120, num_test=1500)
```
you only need to specify the number of training points, and then DeepXDE will automatically set up the training points for you (random uniform in the domain).

If you don't want DeepXDE to do this for you, you can also specify the locations of the training points using:
```
dde.data.PDE(..., anchors=X)
```
where `X` is a `N` by `d` matrix of total `N` points, with each row is one point of `d`-dim.

**Comment by hbadi** [comment]
Hi,
Nive job !
1 - I manage to run this example but the simulation is very slow, it takes 15min, is it normal ?
2 - Is there a way to define multiple geometry domain ? I'd like to solve the heat equation in a decomposed domain \Omega_i (each one have a conductivity coefficient).
3 - When do you plan to switch to tensorflow 2 ?

Best regards.

**Comment by lululxvi** [comment]
@hbadi 

- Which version are you using? Update to the latest version if not.
- Yes, you can define a geometry composed from more than one sub-domains, e.g.,
```
geom1 = dde.geometry.Rectangle(...)
geom2 = dde.geometry.Disk(...)
geom = geom1 | geom2
```
For more details, check Fig 5 in the [paper](https://arxiv.org/abs/1907.04502)
- Hopefully, TF 2 will be supported in one month.

**Comment by hbadi** [comment]
I ran the L-shape laplacian with deepxde installed from pip (version 0.5)

**Comment by lululxvi** [comment]
There is a bug in v0.5.0. You can either upgrade to v0.5.1, or to the latest version 0.6.0. I recommend v0.6.0, but there is an API change in `PDE` class (check the examples).

**Comment by lululxvi** [comment]
TF2 is supported.
